multiload/load-graph.c

```diff
@@ -399,30 +399,30 @@ load_graph_new (MultiloadApplet *ma, guint n, const gchar *label,
     case MATE_PANEL_APPLET_ORIENT_UP:
     case MATE_PANEL_APPLET_ORIENT_DOWN:
     {
-	g->orient = FALSE;
-	break;
+      g->orient = FALSE;
+      break;
     }
     case MATE_PANEL_APPLET_ORIENT_LEFT:
     case MATE_PANEL_APPLET_ORIENT_RIGHT:
     {
-	g->orient = TRUE;
-	break;
+      g->orient = TRUE;
+      break;
     }
     default:
-	g_assert_not_reached ();
+      g_assert_not_reached ();
     }
 
     if (g->show_frame)
     {
-	g->frame = gtk_frame_new (NULL);
```

Comment:
Corrected some of these tab/space mix cases in master - you'll probably need to rebase